kmod/src/server.c

```diff
@@ -439,6 +512,8 @@ static int server_get_log_trees(struct super_block *sb,
 		memset(&lt, 0, sizeof(lt));
 		lt.rid = key.sklt_rid;
 		lt.nr = key.sklt_nr;
+
+		oino_newroot_alloc(sb, &lt.oino_bloom_ref);
```

Comment:
We should check for errors here.  It looks like if (ret < 0) goto unlock; should be fine.